src/tools/run-make-support/src/diff/mod.rs

```diff
@@ -58,15 +66,24 @@ impl Diff {
         self
     }
 
+    pub fn normalize(&mut self, regex: &str, replacement: &str) -> &mut Self {
```

Comment:
```suggestion
    pub fn normalize<R: Into<String>, I: Into<String>>(&mut self, regex: R, replacement: I) -> &mut Self {
```
maybe? so caller don't have to think about `&str`, `String`, ...

Reply:
Good idea!